pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -189,6 +189,15 @@ func getPreFilterState(cycleState *framework.CycleState) (*preFilterState, error
 	return s, nil
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+func (f *Fit) EventsToRegister() []framework.ClusterEvent {
+	return []framework.ClusterEvent{
+		{Resource: framework.Pod, ActionType: framework.Delete},
```

Comment:
What about Update? https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources#scheduler-and-api-server-interaction

Reply:
Yes, update should be added when "in-place-update" is supported in scheduler, also more importantly, we need to update the logic of `AssignedPodUpdated`:

https://github.com/kubernetes/kubernetes/blob/c7fef196b60856420ce3f0470acd1093ab1d9b5f/pkg/scheduler/internal/queue/scheduling_queue.go#L516-L520

So in other words, even we register PodUpdate, it's a no-op as it's blocked by `AssignedPodUpdated`.